components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/overview-trend/overview-trend.component.ts

```diff
@@ -317,12 +322,7 @@ export class OverviewTrendComponent implements OnChanges, OnDestroy  {
   }
 
   createUtcDate(time: string): Date {
-    const utcDate = moment(time).utc();
-    if (utcDate.isValid) {
-      return new Date(utcDate.year(), utcDate.month(), utcDate.date());
-    } else {
-      console.error('Not a valid date ' + time);
-      return new Date();
-    }
+    const utcDate = moment(time);
```

Comment:
```suggestion
    const utcDate = moment.utc(time);
```
We still need to make sure this time is coming back in UTC format.  This is the same as `moment(time).utc()` but a little cleaner to read in my opinion